File: app/models/flight.rb
# == Schema Information
#
# Table name: flights
#
#  id               :integer          not null, primary key
#  name             :text
#  origin_date      :datetime
#  destination_date :datetime
#  origin           :text
#  destination      :text
#  airplane_id      :integer
#  created_at       :datetime
#  updated_at       :datetime
#

class Flight < ActiveRecord::Base

  belongs_to :airplane
  has_many :reservations

end

File: app/assets/javascripts/models/flight.js
var app = app || {};

app.Flight = Backbone.Model.extend({
  urlRoot: '/flights',
  defaults: {
    name: 'Noname',
    origin_date: '1996-12-19T16:39:57-08:00',
    destination_date: '1996-12-19T16:39:57-08:00',
    origin: 'default',
    destination: 'default',
    airplane_id: '0'
    }
});

// newUser = new app.User({name: $('#name').val()});
